### DISCUSSION TITLE
Gpx Track nur teilweise nachfahren

### DISCUSSION THREAD

**Discussion by Micha2345** [osmandapp/OsmAnd#17628] [Maps Related]
Ich nutze Osmand schon länger bin sehr zufrieden damit. Aber ein Problem habe ich bislang noch nicht gelöst:
Ich lade eine gpx Strecke von beispielsweise 100 km Länge. Ich möchte aber nur ein Teilstück befahren.
Es gelingt mir nicht das einzugeben.
Leider nimmt Osmand immer die gesamte gpx Strecke. Wie kann ich dieses Problem lösen?
Herzlichen Dank für eure Hilfe zu diesem Thema, Michael

**Comment by DmitryAlexei** [comment]
If your track consists of multiple segments, you can choose which segment you want to follow (see [link](https://docs.osmand.net/docs/user/navigation/setup/gpx-navigation/#choose-segments)).
You can also choose *navigate to nearest point* in follow [track menu](https://docs.osmand.net/docs/user/navigation/setup/gpx-navigation/#follow-track-menu)
